Formula/man-db.rb

```diff
@@ -0,0 +1,44 @@
+class ManDb < Formula
+  desc "Modern, featureful implementation of the Unix man page system"
+  homepage "https://nongnu.org/man-db/"
+  url "https://download.savannah.nongnu.org/releases/man-db/man-db-2.8.5.tar.xz"
+  sha256 "b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1"
+
+  depends_on "libpipeline"
+
+  def install
+    ENV["CC"] = "#{ENV.cc} -arch x86_64"
```

Comment:
As far as I know the homebrew env already defines this, can you check without?

Reply:
Yep, checked and removed.